CHANGELOG.md

```diff
@@ -3,14 +3,24 @@
 The format is based on [Keep a Changelog](https://keepachangelog.com/en/1.0.0/)
 and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0.html).
 
-## v1.0dev - [date]
+## v1.0.2dev - [11/02/2021]
```

Comment:
We deleted the release, so the changelog must start again at release v1.0.0 - it should reflect the available releases and version numbers in the GitHub repo.

Reply:
Should I just delete the version change history and merge all the change as v1.0.0dev, right?

Reply:
Yup! Or as we're hoping to go for a release right away, maybe 1.0 not dev.